README.md

```diff
@@ -1,35 +1,35 @@
-Ghub.el — Minuscule client library for the Github API
+Ghub.el — Miniscule client library for the GitHub API
 =====================================================
 
-Ghub is a library that provides basic support for using the Github
+Ghub is a library that provides basic support for using the GitHub
 REST (v3) and GraphQL (v4) APIs from Emacs packages.  It abstracts
 access to API resources using only a handful of functions that are
 not resource-specific.
 
-It also handles the creation, storage and use of access tokens using a
-setup wizard, to make it easier for users to get started and to reduce
-the support burden imposed on package maintainers.  It also comes with
+It also handles the creation, storage, and use of access tokens by using a
+setup wizard to make it easier for users to get started and to reduce
+the support burden imposed on package maintainers and comes with
```

Comment:
I agree that two `It also` is bad, and fixed it by replacing the first one with `Ghub`.